omod/src/main/webapp/fragments/exceptionDetails.gsp

```diff
@@ -21,7 +21,7 @@
             <td>${ outgoingMessage.timestamp }</td>
         </tr><tr>
             <th class="label">${ ui.message(artifactId + ".details.failureReason.label") }</th>
-            <td>${ outgoingMessage.failureReason }</td>
+            <td><textarea style="width:100%" disabled>${ outgoingMessage.failureReason }</textarea></td>
```

Comment:
Are you sure that using disabled text area as a failure reason container is a good idea?

Reply:
I will change it to readonly. Readonly doesn't change colours of the field